tasks/go.py

```diff
@@ -428,3 +435,182 @@ def go_fix(ctx, fix=None):
             for osname in oslist:
                 tags = set(ALL_TAGS).union({osname, "ebpf_bindata"})
                 ctx.run(f"go fix{fixarg} -tags {','.join(tags)} ./...")
+
+
+def get_deps(ctx, path):
+    with ctx.cd(path):
+        # Might fail if no mod tidy
+        deps = ctx.run("go list -deps ./...", hide=True, warn=True).stdout.strip().split('\n')
+        prefix = 'github.com/DataDog/datadog-agent/'
+        deps = [
+            dep[len(prefix) :]
```

Comment:
I'd use https://docs.python.org/3.11/library/stdtypes.html#str.removeprefix so you don't have to play with `len` :)